settings.py

```diff
@@ -17,6 +17,10 @@
     }
 }
 
+MIDDLEWARE_CLASSES = (
```

Comment:
It's to avoid this warning: 
```
System check identified some issues:

WARNINGS:
?: (1_7.W001) MIDDLEWARE_CLASSES is not set.
	HINT: Django 1.7 changed the global defaults for the MIDDLEWARE_CLASSES. 
django.contrib.sessions.middleware.SessionMiddleware, 
django.contrib.auth.middleware.AuthenticationMiddleware, and 
django.contrib.messages.middleware.MessageMiddleware were removed from the defaults. 
If your project needs these middleware then you should configure this setting.
```

It just explicitly sets MIDDLEWARE_CLASSES to their (Django 1.7) default value.

It's not relevant to this bug -- it was just annoying. I'll take it out this PR, and we can include it when we upgrade dimagi-utils to Django 1.7.